akka-docs/rst/project/migration-guide-2.4.x-2.5.x.rst

```diff
@@ -440,6 +440,18 @@ They will receive ``Replicator.Deleted`` instead.
 Persistence
 ===========
 
+Binary incompatibility of PersistentActor and AtLeastOneDelivery
+----------------------------------------------------------------
+
+To be able to evolve the Java APIs of ``PersistentActor`` and ``AtLeastOneDelivery`` to work with Scala 2.12
+we could find no other solution but to break the binary compatibility of the Scala versions (which the Java ones
+were based on). T
+
+his means that the Akka 2.5 artifact cannot be a class path drop in replacement of Akka 2.4 and earlierif you use
```

Comment:
I don't think you have to mention earlier because persistence was experimental in 2.3 and changed